early-resolve.js

```diff
@@ -35,11 +34,13 @@ async function getSSMParameter(parameter) {
       if (process.env.TEST) {
         ret = 'mocked';
       } else {
-        const param = await ssm.getParameter({
+        const client = new SSMClient();
```

Comment:
Please initiate the client at the global scope. It provides a slightly cleaner code, and is also recommended by Lambda for better initialization.